tests/plugins/lti_consumer/test_cms_plugins.py

```diff
@@ -0,0 +1,61 @@
+"""Testing DjangoCMS plugin declaration for Richie's lti consumer plugin."""
+from django.test import TestCase, override_settings
+from django.test.client import RequestFactory
+
+from cms.api import add_plugin
+from cms.models import Placeholder
+from cms.plugin_rendering import ContentRenderer
+from django.conf import settings
+
+from richie.plugins.lti_consumer.cms_plugins import LTIConsumerPlugin
+from richie.plugins.lti_consumer.factories import LTIConsumerFactory
+
+
+class LTIConsumerPluginTestCase(TestCase):
+    """Test suite for the lti consumer plugin."""
+
+    @override_settings(
+        LTI_CONSUMER={
+            "lti_message_type": "Marsha Video",
+            "lti_version": "Marsha Video",
+            "resource_link_id": "Marsha Video",
+            "oauth_consumer_key": "InsecureOauthConsumerKey",
+            "oauth_signature_method": "InsecureSharedSecret",
+            "oauth_timestamp": True,
+            "oauth_nonce": True,
+            "oauth_version": 0.5625,
+            "oauth_signature": 0.5625,
+            "oauth_callback": 0.5625,
+        }
+    )
```

Comment:
all this won't come from settings. The setting should get the same parameters as the one in our [LTI configurable xblock](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwinns3b6truAhWTD-wKHSdeAbYQFjAAegQIAhAC&url=https%3A%2F%2Fgithub.com%2Fopenfun%2Fxblock-configurable-lti-consumer&usg=AOvVaw2-e1rNRw2DI97h2_HwLx0D)

About the params listed here:

- "lti_message_type": not sure but could come from the setting
- "lti_version": probably 1.1 since we don't need advanced features
- "resource_link_id": use the plugin instance id?
- "oauth_consumer_key":  comes from a secret setting (see our LTI configurable xblock)
- "oauth_signature_method", "oauth_timestamp", "oauth_nonce", "oauth_version", "oauth_signature": generated when computing the signature